### ISSUE TITLE
NHK World Japan errors on launch

### ISSUE THREAD

**Issue by BuckShotSX** [sbroenne/plugin.video.nhkworldtv#112]
Hi,
Kodi Version 20.5 x64.
NHK World TV 1.1.2

On launch of the plugin it errors out.    I have tried with API cache enabled and disabled.   
I have some shows saved to Favourites, like Document 72 Hours, The Professsionals, etc.   When I go to those Kodi Favourites, I can see the listing of the shows and can play the videos from there.

[Kodi.log](https://github.com/sbroenne/plugin.video.nhkworldtv/files/14765470/Kodi.log)

I have attached the part of the log with the error message.

Thanks a lot for this great plugin, watching NHK will hold me over until I get to visit Japan again.

Arigatō,
Anthony

**Comment by sbroenne** [comment]
Hi Anthony, thanks for reporting this.

NHK World have [redesigned their website](https://www3.nhk.or.jp/nhkworld/upld/thumbnails/en/information/the_nhkworldjapan_websitenow_newlook_20240328.pdf).

Some sections that I used to show have been removed (e.g. playlists). I have to have a deeper look on the new site and adjust the plugin accordingly.

I will probably do an interim release where I simply remove the Playlist menu item. Shouldn't take too long. Thanks for your patience.
